index.html

```diff
@@ -1057,6 +1057,47 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
     <section><h3>The <dfn>check data schema</dfn> algorithm</h3>
       To run the <a>check data schema</a> steps on |payload| and |schema:object|,
       <ol>
+        <li>
+          If |schema| is `null` or `undefined`, return `undefined`.
+        </li>
+        <li>
+          <!--
+            TODO: Should we just continue with the rest of the algorithm here?
+          -->
+          If |schema|.|default| is not |undefined| and |payload| is undefined,
+          let |payload| be the value of |schema|.|default|.
+        </li>
+        <li>
+          Let |const| be |schema|.|const|.
+          If |const| is not |undefined|,
+          <ol>
+            <li>
+              <!-- TODO: Should we already return here? -->
+              If |const| equals |payload|, return |payload|.
+            </li>
+            <li>
+              If |const| does not equal |payload|, throw {{TypeError}} and stop.
+            </li>
+          </ol>
```

Comment:
Corresponding with https://github.com/w3c/wot-scripting-api/pull/534/files#r1502130414, this could become something like this:

```suggestion
          If |schema|.|const| is not |undefined| and |schema|.|const| does not equal |payload|, throw {{TypeError}} and stop.
```

Reply:
FYI: JSON Schema is very unpredictable in a way. The following schema is valid 

```json
{
 "type": "number",
 "const": "abc"
}
```

but there is no possible valid input. Anyhow, the schema itself is valid according to some testing with https://jsonschemalint.com.

Anyhow, your proposal makes sense to me.

Reply:
Thank you for your feedback, @danielpeintner! I will then apply the suggestion above :)